scripts/enable-datastore-viewer.sh

```diff
@@ -96,12 +97,15 @@ server {
     }
 }
 "
-	if [ -e /etc/nginx/sites-enabled/${app_id}.conf ]; then
-		cp /etc/nginx/sites-enabled/${app_id}.conf /tmp
-		echo "$pippo" >> /etc/nginx/sites-enabled/${app_id}.conf
-		echo "Datastore viewer enabled for $app_id at http://$(cat /etc/appscale/my_public_ip):$VIEWER_PORT. Allowed IP: $IP."
+
+if [ -e /etc/nginx/sites-enabled/${APP_ID}.conf ]; then
+		cp /etc/nginx/sites-enabled/${APP_ID}.conf /tmp
+		echo "$pippo" >> /etc/nginx/sites-enabled/${APP_ID}_datastore_viewer.conf
+		echo "Datastore viewer enabled for $APP_ID at http://$(cat /etc/appscale/my_public_ip):$VIEWER_PORT. Allowed IP: $IP."
 		service nginx reload
-	else
-		echo "Cannot find configuration for ${app_id}."
-	fi
-done
+		echo "Note: For a multi node deployment, you will need to forward the admin port from one of the AppServers on another node to the head node"
+        echo "You might need to type in the password for the app engine node."
+        echo "Run the SSH Tunnelling command: ssh -L ${LOCAL_PORT}:localhost:$port $APPENGINE_IP -N"
```

Comment:
It's good practice to use {} around variables when printing in Bash to avoid ambiguities when expanding the variable.

Please add for port and APPENGINE_IP here and throughout.